From fd7b5bd7d18240c55e8a9e1540c8c4da98aff2ee Mon Sep 17 00:00:00 2001
From: Andrei Stefanescu <andrei.stefanescu@nxp.com>
Date: Thu, 12 Jan 2023 18:41:56 +0200
Subject: [PATCH 32/54] fdts: s32cc: add generic compatible for SIUL2 pinctrl
 node

This compatible can be searched in order to switch the pinctrl phandle
to be used by U-Boot when using the SCP firmware.

Issue: ALB-9426

Upstream-Status: Pending 

Signed-off-by: Andrei Stefanescu <andrei.stefanescu@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 fdts/s32g.dtsi   | 5 +++--
 fdts/s32r45.dtsi | 5 +++--
 2 files changed, 6 insertions(+), 4 deletions(-)

diff --git a/fdts/s32g.dtsi b/fdts/s32g.dtsi
index 31bf3d300..dee789fd1 100644
--- a/fdts/s32g.dtsi
+++ b/fdts/s32g.dtsi
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: (GPL-2.0+ OR BSD-3-Clause)
 /*
- * Copyright 2017-2022 NXP
+ * Copyright 2017-2023 NXP
  */
 #include <dt-bindings/clock/s32g-clock.h>
 #include <dt-bindings/clock/s32g-scmi-clock.h>
@@ -108,7 +108,8 @@
 				 <EIRQS_SIUL2_1 0x0 0x0 0x44010010 0x0 0xb4>;
 
 			pinctrl: siul2-pinctrl@4009c240 {
-				compatible = "nxp,s32g-siul2-pinctrl";
+				compatible = "nxp,s32g-siul2-pinctrl",
+					     "nxp,s32cc-siul2-pinctrl";
 				#pinctrl-cells = <2>;
 
 				reg = <MSCRS_SIUL2_0    0 0x0 0x198>,
diff --git a/fdts/s32r45.dtsi b/fdts/s32r45.dtsi
index 3a7db55f5..1c74a5d2b 100644
--- a/fdts/s32r45.dtsi
+++ b/fdts/s32r45.dtsi
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: (GPL-2.0+ OR BSD-3-Clause)
 /*
- * Copyright 2017-2022 NXP
+ * Copyright 2017-2023 NXP
  */
 #include <dt-bindings/clock/s32r45-clock.h>
 #include <dt-bindings/clock/s32r45-scmi-clock.h>
@@ -108,7 +108,8 @@
 				 <EIRQS_SIUL2_1 0 0x0 0x4403c010 0x0 0xb4>;
 
 			pinctrl: siul2-pinctrl@4009c240 {
-				compatible = "nxp,s32r45-siul2-pinctrl";
+				compatible = "nxp,s32r45-siul2-pinctrl",
+					     "nxp,s32cc-siul2-pinctrl";
 				#pinctrl-cells = <2>;
 
 				reg = <MSCRS_SIUL2_0   0 0x0 0x198>,
-- 
2.25.1

